main/main.cpp

```diff
@@ -4766,6 +4768,8 @@ bool Main::iteration() {
 		ScriptServer::get_language(i)->frame();
 	}
 
+	GDExtensionManager::get_singleton()->frame();
```

Comment:
Perhaps this should run before `ScriptServer` does the same? Scripts can be attached to GDExtension objects, so it kind of makes sense to me that GDExtension would be able to go first.